webrtc.html

```diff
@@ -3930,6 +3930,44 @@
         type <code>relay</code>, the <code>url</code> property of the event
         MUST be set to the URL of the ICE server from which the candidate was
         obtained.</p>
+
+        <div class="note">
+          The <code><a>icecandidate</a></code> event is used for three
+          different types of indications:
+          <ul>
+            <li>
+              <p>An candidate has been gathered. The <code><a
```

Comment:
a?

Reply:
I must have gone "an"-crazy from the "an RTC" thing.